app/main/views/sign_in.py

```diff
@@ -139,21 +139,38 @@ def verify_email(user, redirect_url):
 
 
 def _handle_e2e_tests(redirect_url):
-    current_app.logger.warning("E2E TESTS ARE ENABLED.")
-    current_app.logger.warning(
-        "If you are getting a 404 on signin, comment out E2E vars in .env file!"
-    )
-    user = user_api_client.get_user_by_email(os.getenv("NOTIFY_E2E_TEST_EMAIL"))
-    activate_user(user["id"])
-    return redirect(url_for("main.show_accounts_or_dashboard", next=redirect_url))
+    try:
+        current_app.logger.warning("E2E TESTS ARE ENABLED.")
+        current_app.logger.warning(
+            "If you are getting a 404 on signin, comment out E2E vars in .env file!"
+        )
+        user = user_api_client.get_user_by_email(os.getenv("NOTIFY_E2E_TEST_EMAIL"))
+        activate_user(user["id"])
+        return redirect(
+            url_for(
+                "main.show_accounts_or_dashboard",
+                next="EMAIL_IS_OK",
+            )
+        )
+    except Exception as e:
+        stre = str(e)
+        stre = stre.replace(" ", "_")
+        # Trying to get a message back to playwright somehow since we can't see the admin logs
+        return redirect(url_for(f"https://{stre}"))
```

Comment:
Should we leave this error handling in?  Totally fine if the answer is yes!  I am just checking that this isn't in here just for debugging previous issues with getting this working, and it would only add more noise going forward that isn't helpful to us.

If it's still helpful, then totally fine to keep!

Reply:
I think it is helpful to keep this.  Because if there is an error that breaks things, this will be the error.  And it is easier to see it here than in `cf logs notify-api-staging` which someone might forget to run and monitor anyway.